lib/ui/text.dart

```diff
@@ -634,7 +645,7 @@ class ParagraphConstraints {
     return typedOther.width == width;
   }
 
-  int get hashCode => width.hashCode;
+  int get hashCode => hashValues(width);
```

Comment:
There's no reason to go through hashValues for just one value.  The hashCode of the value itself is fine.